packages/valory/services/apy_estimation_hardhat/service.yaml

```diff
@@ -35,8 +35,7 @@ models:
         alpha: 0.25
       pair_ids:
         uniswap_subgraph:
-        - '0x5777d92f208679db4b9778590fa3cab3ac9e2168'
-        - '0xa374094527e1673a86de625aa59517c5de346d32'
```

Comment:
What does this change mean?

Reply:
I had accidentally used some v3 pool ids, so I replaced them with a v2.